logging/cluster-logging-support.adoc

```diff
@@ -9,9 +9,9 @@ toc::[]
 include::snippets/logging-supported-config-snip.adoc[]
 include::snippets/logging-compatibility-snip.adoc[]
 
-The {logging-title} is an opinionated collector and normalizer of application, infrastructure, and audit logs. It is intended to be used for forwarding logs to various supported systems.
+{logging-uc} {for} is an opinionated collector and normalizer of application, infrastructure, and audit logs. It is intended to be used for forwarding logs to various supported systems.
```

Comment:
The variables in this file are not rendering in preview.

Reply:
Fixed!